modules/ROOT/pages/pcf-space-config.adoc

```diff
@@ -31,7 +29,7 @@ python space_configuration_tool.py -i space_settings.yml
 
 == To Configure the space_settings.yml File
 
-In the `space_settings.yml` file, configure the `pcf-services` and `pcf-cloud-params` properties as required. You must create and configure a separate version of this file for each PCF space you are configuring.
+In the `space_settings.yml` file, configure the `pcf-services` and `pcf-cloud-params` properties as required.
 
 === pcf-services
```

Comment:
I think some of the services listed on this table are no longer needed. 
Would need someone from engineering to confirm.